pkg/syncthing/syncthing.go

```diff
@@ -702,32 +668,36 @@ func (s *Syncthing) Restart(ctx context.Context) error {
 }
 
 // Stop halts the background process and cleans up.
-func (s *Syncthing) Stop(force bool) error {
-	pidPath := filepath.Join(s.Home, syncthingPidFile)
-	pid, err := getPID(pidPath)
-	if os.IsNotExist(err) {
-		return nil
-	}
-
-	if !force {
-		if pid != s.pid {
-			log.Infof("syncthing pid-%d wasn't created by this command, skipping", pid)
-			return nil
-		}
-	}
-
-	if err := s.cleanupDaemon(pid, force); err != nil {
+func (s *Syncthing) Stop(wait bool) error {
+	pList, err := process.Processes()
+	if err != nil {
 		return err
 	}
-
-	if err := os.Remove(pidPath); err != nil {
-		if os.IsNotExist(err) {
-			return nil
+	for _, p := range pList {
+		exe, err := p.Exe()
+		if err != nil {
+			log.Infof("error getting procress exec: %s", err.Error())
```

Comment:
add the pid or process name so it helps troubleshoot?
```suggestion
			log.Infof("error getting exec path for process %s: %s", p.Pid, err.Error())
```